sdks/go/pkg/beam/core/runtime/exec/pardo.go

```diff
@@ -148,6 +157,7 @@ func mustExplodeWindows(fn *funcx.Fn, elm *FullValue, usesSideInput bool) bool {
 	return explode || usesSideInput
 }
 
+// FinishBundle finishes processing for this bundle and does post bundle processing operations.
```

Comment:
Same as above comment. "post-bundle-processing"

Reply:
Went without the bundle-processing hyphen, since post-bundle is the adverb for the verb processing.

Reply:
Oh yeah, that looks way better than my suggestion :+1: